services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -465,6 +472,7 @@ private void processSlot() {
       slotEventsChannelPublisher.onSlot(nodeSlot);
       this.currentSlotGauge.set(nodeSlot.longValue());
       this.currentEpochGauge.set(compute_epoch_at_slot(nodeSlot).longValue());
+      this.finalizedEpochGauge.set(recentChainData.getFinalizedEpoch().longValue());
```

Comment:
Strictly speaking we only need to update this on each epoch as finalization only happens at epoch boundaries.